eksrollup/cli.py

```diff
@@ -102,19 +125,15 @@ def scale_up_asg(cluster_name, asg, count):
         else:
             scale_asg(asg_name, asg_old_desired_capacity, desired_capacity, asg_old_max_size)
 
-        cluster_health_wait = app_config['CLUSTER_HEALTH_WAIT']
-        logger.info(f'Waiting for {cluster_health_wait} seconds for ASG to scale before validating cluster health...')
-        time.sleep(cluster_health_wait)
-        asg_instance_count = count_all_cluster_instances(cluster_name, predictive=predictive)
 
```

Comment:
The Travis build is failing due to a minor style nitpick here - two blank lines rather than 1 👍

Reply:
Fixed it!